module/VuFind/src/VuFind/Recommend/LibGuidesProfile.php

```diff
@@ -62,6 +62,34 @@ class LibGuidesProfile implements
      */
     protected $libGuides;
 
+    /**
+     * List of strategies enabled to find a matching LibGuides profile
+     *
+     * @var int
+     */
+    protected $strategies = [];
```

Comment:
I agree that would be a "quicker to get it working" solution but I'd rather force the deployer to choose.  Because 'subject' may work out-of-the-box but it's pretty poor, and I'd hate for someone to miss the fact in config that it can work better.  Also 'subject' always returns a result, so it should likely be disabled if the deployer prefers the other strategy, and having it on by default might make that less obvious.

Reply:
I added an empty($this->strategies) check in getResults(), is that ok?  It seemed cleaner than defining a fake exception-throwing strategy.

Reply:
Yeah, I agree that's more straightforward in this instance!